dev/com.ibm.ws.ssl/resources/com/ibm/ws/ssl/resources/ssl.nlsprops

```diff
@@ -660,4 +660,15 @@ ssl.update.certificate.error.useraction=Ensure the location is accessible by the
 ssl.certificate.add.CWPKI0830I=CWPKI0830I: Certificate with the {0} SubjectDN from the {1} environment variable is being added to the {2} keystore.
 ssl.certificate.add.CWPKI0830I.explanation=A certificate was retrieved from an environment variable in the configuration and is being added to the keystore file.
 ssl.certificate.add.CWPKI0830I.useraction=No action is required.
-# -------------------------------------------------------------------------------------------------
\ No newline at end of file
+# -------------------------------------------------------------------------------------------------
+ssl.protocol.error.CWPKI0831E=CWPKI0831E: The {0} SSL/TLS protocol can not be used.  Extended error is: {1} 
+ssl.protocol.error.CWPKI0831E.explanation:The protocol provided can not be used.  There is an error when trying to get a SSLContext instance with the protocol value. 
```

Comment:
Suggested rewrite of the explanation to simplify the wording and increase the quality check scores:

The protocol cannot be used. An error occurs when trying to get a SSLContext instance with the protocol value.

Reply:
@helyarp The message has been updated please review again.